app/javascript/src/QuickStarts/templates/links.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export default [
+  ['this-text-from-the-yml-will-be-replaced', '/with/this/href', 'And this text in the HTML']
```

Comment:
1. The exact text that will be replaced from the yaml. It will replace the first match, therefore it must be a unique identifier that can't be accidentally reused.
2. The URL used in the anchor tag.
3. The text the user will see in the guide.